vcd/api_test.go

```diff
@@ -72,3 +84,77 @@ func TestTags(t *testing.T) {
 		showTags()
 	}
 }
+
+// Finds the current directory, through the path of this running test
+func getCurrentDir() string {
+	_, currentFilename, _, _ := runtime.Caller(0)
+	return filepath.Dir(currentFilename)
+}
+
+// Reads the version from the VERSION file in the root directory
+func getMajorVersion() string {
```

Comment:
We are duplicating code here. We already have a getMajorVersion() in config_test_.go

Reply:
Not a duplication.
I moved the function to be visible outside tags.
When I implement the `unit` tag, this function should remain in this file to allow visibility across tags.

Reply:
Oops! Missed it. Sorry.